test/functional/testrunner.js

```diff
@@ -54,28 +54,22 @@ const browsers = isSauceLabs
   ? config.desktop.saucelabs
   : config.desktop.browser;
 
-let testcafe;
-createTestCafe()
-  .then(tc => {
-    testcafe = tc;
-    // const runner = testcafe.createRunner();
-    const runner = watch
-      ? testcafe.createLiveModeRunner()
-      : testcafe.createRunner();
-    const testFolder = config.desktop.testsFolder;
-    const testSuite = allFilesSync(testFolder);
-    return (
-      runner
-        .startApp("npm run test:server", 4000)
-        .src(testSuite)
-        .filter(testName => testName.includes(specId))
-        .browsers(browsers)
-        // .reporter("allure")
-        .concurrency(config.desktop.concurrency)
```

Comment:
Did you want to carry over this "concurrency" call? I'm not entirely sure how it works.